CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.12.4
+- Updated getOfferings call to be performed serially instead of concurrently.
```

Comment:
nitpick: maybe we could add backticks to `getOfferings`

Reply:
Good idea, done!